flyway/migrations/V5__Create_SittingJobTable.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TABLE SittingJob (
```

Comment:
Should be called BOOKINGS
See table from ERD:
![image](https://github.com/House-and-Pet-Sitting-Database-Levelup/HouseAndPetSittingDB/assets/125252284/7afc362c-5b16-4a44-8dc0-1720a37c8775)

Reply:
I think I might have looked to the wrong ERD here.
But I will try fixing it. 
And I can see that the branch has been merged?